src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1565,6 +1565,16 @@ protected async Task<DebugStore> RuntimeReady(SessionId sessionId, CancellationT
                 .GroupBy(l => l.Id);
             return locations;
         }
+
+        private static IEnumerable<IGrouping<SourceId, SourceLocation>> GetBPReqNextLocation(DebugStore store, BreakpointRequest req)
```

Comment:
Maybe this should be done in `GetBPReqLocations` itself? Maybe with a `bool ifNoneFoundThenFindNext` param?